src/helix/core/HelixSprite.hx

```diff
@@ -24,6 +25,7 @@ class HelixSprite extends FlxSprite
     public var collisionCallbacks = new Map<FlxBasic, Dynamic->Dynamic->Void>();
     public var textField:FlxText;
     // End internal fields
+    public var keybindMap = new Map<Array<FlxKey>, Void->Void>();

```

Comment:
I don't think we need a map. We just need a single `onKeyDown(Array<FlxKey>->Void)` callback.